spec/support/spec_config.rb

```diff
@@ -92,6 +92,23 @@ def connect_options
     end
   end
 
+  def supports_scram_256?
```

Comment:
This method should be in ClusterConfig which returns information about the deployment. SpecConfig is meant for information about how the test suite was configured.